kmath-histograms/src/jvmMain/kotlin/space/kscience/kmath/histogram/TreeHistogramSpace.kt

```diff
@@ -26,30 +29,35 @@ private fun <B : ClosedFloatingPointRange<Double>> TreeMap<Double, B>.getBin(val
     return null
 }
 
-@UnstableKMathAPI
+public data class ValueAndError(val value: Double, val error: Double)
```

Comment:
Some documentation for this class would be nice.

Reply:
Good point. Maybe some additional refacoring is in order.